Fun.Blazor/DslCE.fs

```diff
@@ -12,13 +12,118 @@ open Microsoft.AspNetCore.Components.Web
 type FunBlazorBuilderWithDomAttrs<'Component when 'Component :> Microsoft.AspNetCore.Components.IComponent> () =
     inherit FunBlazorBuilder<'Component>()
 
-    [<CustomOperation("childContent")>] member this.childContent (_: FunBlazorBuilder<'Component>, v: Node list) = this.AddNodes v
-    [<CustomOperation("childContent")>] member this.childContent (_: FunBlazorBuilder<'Component>, v: string) = Text v |> this.AddNode
-    [<CustomOperation("childContent")>] member this.childContent (_: FunBlazorBuilder<'Component>, v: int) = Text (string v) |> this.AddNode
-    [<CustomOperation("childContent")>] member this.childContent (_: FunBlazorBuilder<'Component>, v: float) = Text (string v) |> this.AddNode
-    [<CustomOperation("childContentRaw")>] member this.childContentRaw (_: FunBlazorBuilder<'Component>, v: string) = RawHtml v |> this.AddNode
+    /// <summary>
+    /// Multiple Nodes that are going to be added to the element's children
+    /// </summary>
+    /// <example>
+    /// <code lang="fsharp">
+    /// // &lt;div>
+    /// //   &lt;p>&lt;/p>
+    /// //   &lt;p>&lt;p/>
+    /// // &lt;/div>
+    /// div() {
+    ///   childContent [ p() {}; p() {} }
+    /// }
+    /// </code>
+    /// </example>
+    [<CustomOperation("childContent")>]
+    member this.childContent (_: FunBlazorBuilder<'Component>, v: Node list) = this.AddNodes v
+    /// <summary>
+    /// Single child node to be added into the element's children
+    /// </summary>
+    /// <example>
+    /// <code lang="fsharp">
+    /// // &lt;div>
+    /// //   &lt;p>This is my content&lt;/p>
+    /// // &lt;/div>
+    /// let myText content =
+    ///   p() {
+    ///    class' "my-class"
+    ///    childContent content
+    ///   }
+    /// div() {
+    ///   childContent (myText "This is my content")
+    /// }
+    /// </code>
+    /// </example>
+    [<CustomOperation("childContent")>]
+    member this.childContent (_: FunBlazorBuilder<'Component>, v: Node) = this.AddNode v
```

Comment:
I added also this overload I think it's useful when you are calling a function to get the node content like in the example

Reply:
please let me know if it is ok or I should remove it

Reply:
Looks great!